tasks/4.js

```diff
@@ -0,0 +1,25 @@
+var arr = [1, 2, 1, 3, 4];
+
+function mapArr(arr) {
+  var newArr = [];
+  arr.map(function (x) {
+    if (!newArr.includes(x)) {
+      newArr.push(x);
+    }
+  });
+
+  return newArr;
+}
+
+function filterArr(arr) {
+  var newArr = arr.filter(function (a, b) {
+    return arr.indexOf(a) === b;
+  });
+
+  return newArr;
+}
```

Comment:
I think it would break if you'd provide `[10, 10, 10]`

Reply:
Nope, works fine.